README.md

```diff
@@ -142,6 +145,16 @@ Review these [**Filters**](#filters) to learn more about limiting access to the
 
 The Block Data API [uses server-side registered blocks][wordpress-block-metadata-php-registration] to determine block attributes. Refer to the **[Client-side blocks](#client-side-blocks)** section for more information about client-side block support limitations.
 
+### Versioning
+
+The current REST endpoint uses a `v1` prefix:
+
+```
+/wp-json/vip-block-data-api/v1/...
+```
+
+We plan to utilize API versioning to avoid unexpected changes to the plugin. In the event that we make breaking changes to API output, we will add a new endpoint (e.g. `/wp-json/vip-block-data-api/v2/`) with access to new data. We intend to keep backwards compatibility with existing version endpoints.
```

Comment:
"existing version endpoints." reads oddly to me, though I don't know why.

"Previous versions will remain accessible for backward compatibility."

Reply:
Much better, thank you!